osu.Game/Database/BeatmapDatabase.cs

```diff
@@ -188,7 +219,7 @@ public void Import(string path)
                 }
             }
 
-            Import(new[] { beatmapSet });
+            return beatmapSet;
         }
 
         public void Import(IEnumerable<BeatmapSetInfo> beatmapSets)

```

Comment:
I left it as-is because it was used in `TestCasePlaySongSelect`, but if desired I can combine it with `Import(paths)`.